.buildkite/scripts/dockerhub_publish.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -euo pipefail
+
+source .buildkite/scripts/shared.sh
+version=$(buildkite-agent meta-data get "version")
+
+if [[ "${FAKE_RELEASE_TAG}" ]]; then
+  export IMAGE_NAME="habitat/fakey-mc-fake-face-studio"
+fi
+
+pushd ./components/rootless_studio >/dev/null
+docker pull "${IMAGE_NAME}:${version}"
```

Comment:
Where does `IMAGE_NAME` come from if we're doing a real release?

Reply:
https://github.com/habitat-sh/habitat/pull/5401/files#diff-0b94ebc237e784dd28240fc35ae17e26R4